### ISSUE TITLE
Crandall - ngsQC and assemblyQC for Listeria monocytogenes - FDA ARGOS BioProject

### ISSUE THREAD

**Issue by Jgergely11** [FDA-ARGOS/data.argosdb#318]
- [x] Please complete ngsQC for the following organisms and all SRRs associated with each Biosample using the HIVE ngsQC protocol.
- [x] Please complete assemblyQC for the following genome assembly IDs
- [ ] Once data is ready to be added to datasets, Check with @Jgergely11 for ngsQC and @penningtonea for assemblyQC

<html xmlns:v="urn:schemas-microsoft-com:vml"
xmlns:o="urn:schemas-microsoft-com:office:office"
xmlns:x="urn:schemas-microsoft-com:office:excel"
xmlns="http://www.w3.org/TR/REC-html40">

<head>

<meta name=ProgId content=Excel.Sheet>
<meta name=Generator content="Microsoft Excel 15">
<link id=Main-File rel=Main-File
href="file:///C:/Users/Home/AppData/Local/Temp/msohtmlclip1/01/clip.htm">
<link rel=File-List
href="file:///C:/Users/Home/AppData/Local/Temp/msohtmlclip1/01/clip_filelist.xml">
<style>

</style>
</head>

<body link="#0563C1" vlink="#954F72">

Organism | Bioproject | Biosample | Genome Assembly ID
-- | -- | -- | --
Listeria monocytogenes | PRJNA231221 | SAMN16357249 | GCA_016728025.1
Listeria monocytogenes | PRJNA231221 | SAMN03255437 | GCA_001188045.2
Listeria monocytogenes | PRJNA231221 | SAMN11056493 | GCA_006364655.1
Listeria monocytogenes | PRJNA231221 | SAMN10346771 | GCA_006364035.1
Listeria monocytogenes | PRJNA231221 | SAMN10163247 | GCA_003812565.1
Listeria monocytogenes | PRJNA231221 | SAMN10163248 | GCA_003812545.1
Listeria monocytogenes | PRJNA231221 | SAMN03255438 | GCA_001188025.2
Listeria monocytogenes | PRJNA231221 | SAMN06173364 | GCA_002206565.2
Listeria monocytogenes | PRJNA231221 | SAMN07312449 | GCA_002557645.1

</body>

</html>

**Comment by steph-sing** [comment]
Additional QC is on hold for the time being. Moving to the backlog.

**Comment by penningtonea** [comment]
**REFERENCE GENOME INFORMATION**
Infraspecific name: Strain: FDAARGOS_1080
BioSample: SAMN16357249
BioProject: PRJNA231221

Reference proteome: UP000000817 
Reference assembly: GCA_000196035.1
SRA files: SRR12933226, SRR12933227

**Comment by steph-sing** [comment]
@Keillyzhang and @emdelattre checking in - how's your progress? Any issues with HIVE or info we provided?

**Comment by steph-sing** [comment]
Complete
